configs/components/echam/echam.datasets.yaml

```
@@ -115,11 +121,13 @@ forcing_sources:
                 "${forcing_dir}/ozone/${resolution}_ozone_historical_@YEAR@.nc":
                         from: 1851
                         to: 2014
-                # scenario: ssp126, ssp119, ssp245, ssp370, ssp585
-                "${forcing_dir}/ozone/${resolution}_ozone_${scenario}_@YEAR@.nc":
+                # Note: The variable ``ozone_source_file_scenario`` is set to ${scenario} by default in line 14.
```

```suggestion
                # Note: The variable ``ozone_source_file_scenario`` is set to ${scenario} at the beginning of the file.
```
Lines are likely to change and we cannot keep track of them